### PR TITLE
270 wcs versioning

### PR DESCRIPTION
**Pull request by daveatweaviate**
[issue](https://github.com/weaviate/weaviate-edu-team-tasks/issues/270)
[staging](https://270-wcs-create-versioning-page--tangerine-buttercream-20c32f.netlify.app/developers/wcs/platform/version)

### CODE REVIEW

File: developers/wcs/platform/version.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Weaviate does not automatically update existing clusters when a new version of Weaviate core is released. 
+
+To check if you are running the latest version of Weaviate core, follow these steps. 
+
+1. Connect to the WCS console, and find your cluster in the Dashboard tab.
+1. Click the "Details" button.
+1. Find the version tile in the details panel. 
+
+The tile has an "Update!" button when a newer version is available. Consider you business needs, and upgrade your cluster when you have a suitable maintenance window.
```
Comment (mikewyer) [inline developers/wcs/platform/version.mdx:23]:
```suggestion
The tile has an "Update!" button when a newer version is available. Consider your business needs, and upgrade your cluster when you have a suitable maintenance window.  
```

File: developers/wcs/platform/version.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+1. Connect to the WCS console, and find your cluster in the Dashboard tab.
+1. Click the "Details" button.
+1. Find the version tile in the details panel. 
+
+The tile has an "Update!" button when a newer version is available. Consider you business needs, and upgrade your cluster when you have a suitable maintenance window.  
+
+## Security upgrades
+
+ Weaviate does not automatically update clusters, however if an existing cluster has a security vulnerability, Weaviate upgrades the cluster when a fix is released.
```
Comment (mikewyer) [inline developers/wcs/platform/version.mdx:27]:
```suggestion
 WCS does not automatically update clusters, however if an existing cluster has a security vulnerability, WCS upgrades the cluster when a fix is released.
```

File: developers/wcs/platform/version.mdx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+ 
+ For security updates, Weaviate upgrades the cluster to the most current production release unless there are known compatibility issues. If there are compatibility issues, Weaviate upgrades the cluster to the most recent compatible version.
```
Comment (mikewyer) [inline developers/wcs/platform/version.mdx:29]:
```suggestion
 For security updates, WCS upgrades the cluster to the most current production release unless there are known compatibility issues. If there are compatibility issues, WCS upgrades the cluster to the most recent compatible version. 
```

File: developers/wcs/platform/version.mdx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+ Weaviate does not automatically update clusters, however if an existing cluster has a security vulnerability, Weaviate upgrades the cluster when a fix is released.
+ 
+ For security updates, Weaviate upgrades the cluster to the most current production release unless there are known compatibility issues. If there are compatibility issues, Weaviate upgrades the cluster to the most recent compatible version. 
+
+## Additional considerations
+
+- **Billing**. If your cluster's Weaviate core version cannot support the billing requirements for your pricing plan, Weaviate upgrades your cluster to the current production version or the most recent compatible version. 
+- **Data integrity**. If your cluster's Weaviate core version has a known risk of data loss or corruption, Weaviate upgrades the cluster when a fix is released.
+- **Downgrade**. You cannot downgrade your WCS cluster to an earlier version of Weaviate core.
+- **Enterprise customers**. If you are an enterprise customer and you require a specific version of Weaviate core, contact your support representative. In some circumstances, Weaviate can install a specific Weaviate core version.
```
Comment (mikewyer) [inline developers/wcs/platform/version.mdx:36]:
```suggestion
- **Billing**. If your cluster's Weaviate core version cannot support the billing requirements for your pricing plan, WCS upgrades your cluster to the current production version or the most recent compatible version. 
- **Data integrity**. If your cluster's Weaviate core version has a known risk of data loss or corruption, WCS upgrades the cluster when a fix is released.
- **Downgrade**. You cannot downgrade your WCS cluster to an earlier version of Weaviate core.
- **Enterprise customers**. If you are an enterprise customer and you require a specific version of Weaviate core, contact your support representative. In some circumstances, WCS can install a specific Weaviate core version.
```

**Review by mikewyer — COMMENTED** [review]
I've suggested using "WCS" where we are referring to the WCS platform and support team. Otherwise it sounds like "weaviate the database" is doing things by itself.

File: developers/wcs/platform/version.mdx (hunk 1 above)
Comment (daveatweaviate) [reply developers/wcs/platform/version.mdx]: Updated

File: developers/wcs/platform/version.mdx (hunk 2 above)
Comment (daveatweaviate) [reply developers/wcs/platform/version.mdx]: Update throughout

File: developers/wcs/platform/version.mdx (hunk 3 above)
Comment (daveatweaviate) [reply developers/wcs/platform/version.mdx]: Updated

File: developers/wcs/platform/version.mdx (hunk 4 above)
Comment (daveatweaviate) [reply developers/wcs/platform/version.mdx]: Updated

**Review by daveatweaviate — COMMENTED** [review]
@mikewyer thanks for your feedback.

**Review by mikewyer — APPROVED** [review]
LGTM!

**Pull request merged**